src/snowflake/cli/api/project/schemas/entities/entities.py

```diff
@@ -20,20 +20,35 @@
 from snowflake.cli.api.entities.application_package_entity import (
     ApplicationPackageEntity,
 )
+from snowflake.cli.api.entities.snowpark_entity import FunctionEntity, ProcedureEntity
 from snowflake.cli.api.entities.streamlit_entity import StreamlitEntity
 from snowflake.cli.api.project.schemas.entities.application_entity_model import (
     ApplicationEntityModel,
 )
 from snowflake.cli.api.project.schemas.entities.application_package_entity_model import (
     ApplicationPackageEntityModel,
 )
+from snowflake.cli.api.project.schemas.entities.snowpark_entity import (
+    FunctionEntityModel,
+    ProcedureEntityModel,
+)
 from snowflake.cli.api.project.schemas.entities.streamlit_entity_model import (
     StreamlitEntityModel,
 )
 
-Entity = Union[ApplicationEntity, ApplicationPackageEntity, StreamlitEntity]
+Entity = Union[
+    ApplicationEntity,
+    ApplicationPackageEntity,
+    StreamlitEntity,
+    ProcedureEntity,
+    FunctionEntity,
+]
```

Comment:
`SnowparkEntity`?

Reply:
Again - `SnowparkEntity` is not meant to be used by user